chapters/Chapter 1 — Untitled.md

```diff
@@ -49,14 +49,76 @@ Something wet was pooling at the bottom of her helmet’s goggles. Right: she wa
 
 With the gravity field she should be good to run, but she kept her pace steady as she moved through her bedroom to the door of the hold, carefully ducking under the flying crate and batteries.
 
-Her hand reached for her gun as she opened the door, but there was no need. Her attacker was tumbling. His neck wasn’t broken yet: he was still managing to bring his arms up in time to protect himself, but he was tiring embarrassingly quickly. A few of his many weapons had detached themselves from him and were now flying around the hold with him.
+Her hand reached for her gun as she opened the door, but there was no need. Her attacker was tumbling. His neck wasn’t broken yet: he was still managing to bring his arms long up in time to protect himself, but he was tiring embarrassingly quickly. A few of his many weapons had detached themselves from him and were now flying around the hold with him.
```

Comment:
Also, I would like to note more characteristics of characters that are based in their own choices. For example, what does this assassin wear? What does he do with his hair? He's in a spacesuit, right? Is it a heavy-duty one for a longer, higher-risk walk? Or a lighter one for better movement?